lib/taurus/qt/qtgui/display/tauruslabel.py

```diff
@@ -118,8 +119,8 @@ def _updateToolTip(self, label):
         toolTip = label.getFormatedToolTip()
         if self._trimmedText:
             toolTip = u"<p><b>Value:</b> %s</p><hr>%s" %\
-                      (str(self._text, errors='replace'),
-                       str(str(toolTip), errors='replace'))
+                      (to_str(self._text, errors='replace'),
+                       to_str(toolTip, errors='replace'))
```

Comment:
Why is this necessary? 
AFAIK, self._text and toolTip should never be of type bytes, right?